packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/service/service.controller.js

```diff
@@ -31,10 +31,10 @@ export default class XdslModemServiceCtrl {
       .get({
         xdslId: this.$stateParams.serviceName,
       })
-      .$promise.then(({ xdslTask }) => {
+      .$promise.then(({ data }) => {
         blocIP.isDefined = true;
-        blocIP.value = xdslTask === 'enabled';
-        return xdslTask;
+        blocIP.value = data === 'enabled';
```

Comment:
Please store `'enabled'` in a constant

Reply:
I note this feedback for a future improvment